query/influxql/transpiler.go

```diff
@@ -691,27 +664,75 @@ func (t *transpilerState) from(m *influxql.Measurement) (ast.Expression, error)
 							Name: "bucket",
 						},
 						Value: &ast.StringLiteral{
-							Value: fmt.Sprintf("%s/%s", db, rp),
+							Value: t.config.Bucket,
 						},
 					},
 				},
 			},
 		}
 	} else {
-		// use mapping bucket id
-		args = []ast.Expression{
-			&ast.ObjectExpression{
-				Properties: []*ast.Property{
-					{
-						Key: &ast.Identifier{
-							Name: "bucketID",
+		if t.dbrpMappingSvc == nil {
+			return nil, errors.New("cannot transpiler, no dbrp mappings provided")
```

Comment:
prolly: influxdb.Error{Msg: "unable to transpile: db and rp mappings need to be created by some way", Code:  is this internal?}